app/models/package_manager/base.rb

```diff
@@ -88,35 +88,8 @@ def self.platform_name(platform)
     end
 
     def self.map_project(project)
-      mapped_project = mapping(project)
-      mapped_project = mapped_project.delete_if { |_key, value| value.blank? } if mapped_project.present?
-      mapped_project
-    end
-
-    def self.save(project, sync_versions: true)
```

Comment:
There's not really a reason for this to be outside of the `update()` method, so I consolidated them.